qCC/ccColorFromScalarDlg.cpp

```diff
@@ -221,7 +217,10 @@ void ccColorFromScalarDlg::updateColormaps()
 		m_colors[0]->clear();
 
 		ccColor::Rgb col = ccColor::Convert::hsv2rgb(360.0f, 1.0f, 1.0f);
-		m_colors[0]->insert(ccColorScaleElement(1.0, QColor(int(col.r), int(col.g), int(col.b), 255)));
+		m_colors[0]->insert(ccColorScaleElement(1.0, QColor(
```

Comment:
You can directly use ccColor::Rgba(col, ccColor::MAX)

Reply:
ccColorScaleElement doesn't seem to have a constructor for (double, ccColor::Rgba) - the only one I can find is for (double, QColor)?

Reply:
Oops sorry, I thought it was the same as the others. At least you don't need to explicitly cast to 'int' the values here.